deployment/address_book.go

```diff
@@ -256,3 +256,18 @@ func SearchAddressBook(ab AddressBook, chain uint64, typ ContractType) (string,
 
 	return "", fmt.Errorf("not found")
 }
+
+func SearchAddressBookForAddress(ab AddressBook, chain uint64, addrToFind string) error {
```

Comment:
It just asserts the presence of address, how about?
```suggestion
func IsAddressPresentInAddressBook(ab AddressBook, chain uint64, addrToFind string) error {
```